template/client/components/Comment.vue

```diff
@@ -117,7 +117,8 @@ export default class Comment extends Vue {
   left: 50%;
   transform: translateX(-50%);
 
-  .number { // this will be available as `this.$style.number`
+  .number {
+    // this will be available as `this.$style.number`
```

Comment:
New line comments starts with uppercase letters.
This is a linting issue: https://github.com/wemake-services/wemake-vue-template/issues/494

Thanks for discovering that!